tests/runner/check_serial_runner.py

```diff
@@ -51,8 +52,9 @@ def check_simple_run(self):
 
         results = runner.run_all_tests()
         assert len(results) == 1
-        assert results.num_failed() == 0
-        assert results.num_passed() == 1
+        assert results.num_failed == 0
+        assert results.num_passed == 1
+        assert results.num_ignored == 0
```

Comment:
Do you want a test that checks that the serial test runner handles ignored tests properly (i.e. validates the test isn't invoked) and gives the right count?